2024-05-04 13:49:38,059 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] WARN  [io.agr.pool] (agroal-21) Datasource '<default>': ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=vpGykRWpSlS831un8szAAQ==)
https://docs.oracle.com/error-help/db/ora-12541/
2024-05-04 13:49:38,060 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] WARN  [org.hib.eng.jdb.env.int.JdbcEnvironmentInitiator] (JPA Startup Thread) HHH000342: Could not obtain connection to query metadata: java.lang.NullPointerException: Cannot invoke "org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(java.sql.SQLException, String)" because the return value of "org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.sqlExceptionHelper()" is null
	at org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.doTheWork(JtaIsolationDelegate.java:186)
	at org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.lambda$delegateWork$1(JtaIsolationDelegate.java:75)
	at org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.doInSuspendedTransaction(JtaIsolationDelegate.java:107)
	at org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.delegateWork(JtaIsolationDelegate.java:72)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.getJdbcEnvironmentUsingJdbcMetadata(JdbcEnvironmentInitiator.java:276)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:107)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:68)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:130)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:263)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:238)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:52)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:136)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:247)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.boot.internal.SessionFactoryOptionsBuilder.<init>(SessionFactoryOptionsBuilder.java:276)
	at io.quarkus.hibernate.orm.runtime.recording.PrevalidatedQuarkusMetadata.buildSessionFactoryOptionsBuilder(PrevalidatedQuarkusMetadata.java:70)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:84)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
	at java.base/java.lang.Thread.run(Thread.java:1583)

2024-05-04 13:49:38,093 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] WARN  [io.agr.pool] (agroal-21) Datasource '<default>': ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=CY/evVtISy2KJQRLxpBJzQ==)
https://docs.oracle.com/error-help/db/ora-12541/
2024-05-04 13:49:38,094 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] ERROR [org.hib.que.sqm.mut.int.tem.GlobalTemporaryTableStrategy] (JPA Startup Thread) Unable obtain JDBC Connection: java.sql.SQLException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=CY/evVtISy2KJQRLxpBJzQ==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1398)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:991)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: oracle.net.ns.NetException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=CY/evVtISy2KJQRLxpBJzQ==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:311)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more

2024-05-04 13:49:38,102 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] WARN  [io.agr.pool] (agroal-21) Datasource '<default>': ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=o5jv6o/rTaSE9R4uYnFwBA==)
https://docs.oracle.com/error-help/db/ora-12541/
2024-05-04 13:49:38,104 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] ERROR [org.hib.que.sqm.mut.int.tem.GlobalTemporaryTableStrategy] (JPA Startup Thread) Unable obtain JDBC Connection: java.sql.SQLException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=o5jv6o/rTaSE9R4uYnFwBA==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1398)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:991)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: oracle.net.ns.NetException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=o5jv6o/rTaSE9R4uYnFwBA==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:311)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more

2024-05-04 13:49:38,110 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] WARN  [io.agr.pool] (agroal-21) Datasource '<default>': ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=oKsPVwjrSqqeCV1ogNovIg==)
https://docs.oracle.com/error-help/db/ora-12541/
2024-05-04 13:49:38,111 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] ERROR [org.hib.que.sqm.mut.int.tem.GlobalTemporaryTableStrategy] (JPA Startup Thread) Unable obtain JDBC Connection: java.sql.SQLException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=oKsPVwjrSqqeCV1ogNovIg==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1398)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:991)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: oracle.net.ns.NetException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=oKsPVwjrSqqeCV1ogNovIg==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:311)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more

2024-05-04 13:49:38,117 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] WARN  [io.agr.pool] (agroal-21) Datasource '<default>': ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=HagPrj4bQi+QevRrXF82sQ==)
https://docs.oracle.com/error-help/db/ora-12541/
2024-05-04 13:49:38,118 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] ERROR [org.hib.que.sqm.mut.int.tem.GlobalTemporaryTableStrategy] (JPA Startup Thread) Unable obtain JDBC Connection: java.sql.SQLException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=HagPrj4bQi+QevRrXF82sQ==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1398)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:991)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: oracle.net.ns.NetException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=HagPrj4bQi+QevRrXF82sQ==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:311)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more

2024-05-04 13:49:38,170 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] WARN  [io.agr.pool] (agroal-21) Datasource '<default>': ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=JyrTaA6pTiK9K7l6AdeZCw==)
https://docs.oracle.com/error-help/db/ora-12541/
2024-05-04 13:49:38,171 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread) SQL Error: 12541, SQLState: 66000
2024-05-04 13:49:38,172 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] ERROR [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread) ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=JyrTaA6pTiK9K7l6AdeZCw==)
https://docs.oracle.com/error-help/db/ora-12541/
2024-05-04 13:49:38,175 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] ERROR [io.qua.run.Application] (Quarkus Main Thread) Failed to start application (with profile [dev]): java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
	at io.quarkus.runtime.Application.start(Application.java:101)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:111)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:71)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:44)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:124)
	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.runner.bootstrap.StartupActionImpl$1.run(StartupActionImpl.java:113)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: jakarta.persistence.PersistenceException: [PersistenceUnit: <default>] Unable to build Hibernate SessionFactory
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:129)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:89)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
	... 1 more
Caused by: org.hibernate.exception.GenericJDBCException: Unable to open JDBC Connection for DDL execution [ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=JyrTaA6pTiK9K7l6AdeZCw==)
https://docs.oracle.com/error-help/db/ora-12541/] [n/a]
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:63)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:94)
	at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.getIsolatedConnection(DdlTransactionIsolatorJtaImpl.java:77)
	at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.getIsolatedConnection(DdlTransactionIsolatorJtaImpl.java:67)
	at org.hibernate.tool.schema.internal.exec.ImprovedExtractionContextImpl.getJdbcConnection(ImprovedExtractionContextImpl.java:63)
	at org.hibernate.tool.schema.extract.spi.ExtractionContext.getQueryResults(ExtractionContext.java:43)
	at org.hibernate.tool.schema.extract.internal.SequenceInformationExtractorLegacyImpl.extractMetadata(SequenceInformationExtractorLegacyImpl.java:39)
	at org.hibernate.tool.schema.extract.internal.DatabaseInformationImpl.initializeSequences(DatabaseInformationImpl.java:66)
	at org.hibernate.tool.schema.extract.internal.DatabaseInformationImpl.<init>(DatabaseInformationImpl.java:60)
	at org.hibernate.tool.schema.internal.Helper.buildDatabaseInformation(Helper.java:185)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:98)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:286)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:145)
	at java.base/java.util.HashMap.forEach(HashMap.java:1429)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:142)
	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:315)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:87)
	... 6 more
Caused by: java.sql.SQLException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=JyrTaA6pTiK9K7l6AdeZCw==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1398)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:991)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	... 1 more
Caused by: oracle.net.ns.NetException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=JyrTaA6pTiK9K7l6AdeZCw==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:311)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more

2024-05-04 13:49:38,194 sebastian-virtual-machine sistema-hoteles-be-dev.jar[5457] ERROR [io.qua.dep.dev.IsolatedDevModeMain] (vert.x-worker-thread-4) Failed to start quarkus: io.quarkus.dev.appstate.ApplicationStartException: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.dev.appstate.ApplicationStateNotification.waitForApplicationStart(ApplicationStateNotification.java:58)
	at io.quarkus.runner.bootstrap.StartupActionImpl.runMainClass(StartupActionImpl.java:132)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.restartApp(IsolatedDevModeMain.java:192)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.restartCallback(IsolatedDevModeMain.java:173)
	at io.quarkus.deployment.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:541)
	at io.quarkus.deployment.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:441)
	at io.quarkus.vertx.http.runtime.devmode.VertxHttpHotReplacementSetup$5.call(VertxHttpHotReplacementSetup.java:150)
	at io.quarkus.vertx.http.runtime.devmode.VertxHttpHotReplacementSetup$5.call(VertxHttpHotReplacementSetup.java:137)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:177)
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:276)
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:209)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1512)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
	at io.quarkus.runtime.Application.start(Application.java:101)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:111)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:71)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:44)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:124)
	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.runner.bootstrap.StartupActionImpl$1.run(StartupActionImpl.java:113)
	... 1 more
Caused by: jakarta.persistence.PersistenceException: [PersistenceUnit: <default>] Unable to build Hibernate SessionFactory
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:129)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:89)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
	... 1 more
Caused by: org.hibernate.exception.GenericJDBCException: Unable to open JDBC Connection for DDL execution [ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=JyrTaA6pTiK9K7l6AdeZCw==)
https://docs.oracle.com/error-help/db/ora-12541/] [n/a]
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:63)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:94)
	at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.getIsolatedConnection(DdlTransactionIsolatorJtaImpl.java:77)
	at org.hibernate.resource.transaction.backend.jta.internal.DdlTransactionIsolatorJtaImpl.getIsolatedConnection(DdlTransactionIsolatorJtaImpl.java:67)
	at org.hibernate.tool.schema.internal.exec.ImprovedExtractionContextImpl.getJdbcConnection(ImprovedExtractionContextImpl.java:63)
	at org.hibernate.tool.schema.extract.spi.ExtractionContext.getQueryResults(ExtractionContext.java:43)
	at org.hibernate.tool.schema.extract.internal.SequenceInformationExtractorLegacyImpl.extractMetadata(SequenceInformationExtractorLegacyImpl.java:39)
	at org.hibernate.tool.schema.extract.internal.DatabaseInformationImpl.initializeSequences(DatabaseInformationImpl.java:66)
	at org.hibernate.tool.schema.extract.internal.DatabaseInformationImpl.<init>(DatabaseInformationImpl.java:60)
	at org.hibernate.tool.schema.internal.Helper.buildDatabaseInformation(Helper.java:185)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:98)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:286)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:145)
	at java.base/java.util.HashMap.forEach(HashMap.java:1429)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:142)
	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:315)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:87)
	... 6 more
Caused by: java.sql.SQLException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=JyrTaA6pTiK9K7l6AdeZCw==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.jdbc.driver.T4CConnection.handleLogonNetException(T4CConnection.java:1398)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:991)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	... 1 more
Caused by: oracle.net.ns.NetException: ORA-12541: Cannot connect. No listener at host 35.211.214.127 port 1521. (CONNECTION_ID=JyrTaA6pTiK9K7l6AdeZCw==)
https://docs.oracle.com/error-help/db/ora-12541/
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:311)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more

